contracts/core/Allo.sol

```diff
@@ -79,8 +79,9 @@ contract Allo is
     /// @dev 'Pool.id' -> 'Pool'
     mapping(uint256 => Pool) private pools;
 
-    /// @custom:oz-upgrades-renamed-from cloneableStrategies
-    mapping(address => bool) private _unusedSlot;
+    /// @notice The trusted forwarder contract address
+    /// @dev Based on ERC2771ContextUpgradeable OZ contracts
+    address private _trustedForwarder;
```

Comment:
^ wouldn't this break upgradeability ? 
Also would it make sense to have multiple trusted forwarded ?

Reply:
> ^ wouldn't this break upgradeability ?

It would not break it, but can introduce problems and confusions in the future. Given this I re-introduced the mapping and added a new variable.

> Also would it make sense to have multiple trusted forwarded ?

I dont think so. At least Biconomy trusted forwarder is open source and callable by anyone. There is no reason to have another one. Did you have something else in mind?

Reply:
No specific reason beyond being able to use another trusted forwarder if something better comes along.
But given that we have the ability to update the forwarder , maybe it doesn't really butter